app/src/main/res/values-es/help.xml

```diff
@@ -222,24 +222,24 @@
     ]]></string>
 
     <string name="help_create_ledger"><![CDATA[
-        <h1>Create Wallet - Ledger</h1>
-        <p>You want to recover your wallet from your Ledger Nano S device.</p>
-        <p>Your secret keys never leave the Ledger device, so you need it plugged in every
-        time you want to access your wallet.</p>
-        <p>Enter a unique wallet name and password. The password is used for securing your wallet data on the Android
-        device. Use a strong password - even better use a passphrase.</p>
-        <p>Enter the block number of the first transaction used for this address in the
-        field \"Restore Height\". You can also use a date in the format YYYY-MM-DD. If you are not sure,
-        enter an approximate date/blockheight <em>before</em> you first used this wallet address.</p>
+        <h1>Crear Monedero - Ledger</h1>
+        <p>Deseas recuperar tu monedero desde tu dispositivo Ledger Nano S.</p>
+        <p>Tus claves secretas nunca dejan el dispositivo Ledger, así que lo necesitas conectado cada
+        vez que deseas acceder a tu monedero.</p>
+        <p>Ingresa un nombre de monedero y contraseña únicos. La contraseña es usada para asegurar tu información del monedero en el dispositivo
+        Android. Usa una contraseña fuerte - o mejor aún, usa una contraseña de frase.</p>
+        <p>Ingresa el número de bloque de la primera transacción usada para esta dirección en el
```

Comment:
Suggestion:
"Ingresa el número **del** bloque..."

Reply:
Got it.